src/libraries/Native/Unix/System.Native/CMakeLists.txt

```diff
@@ -74,10 +74,13 @@ if (GEN_SHARED_LIB)
     install_with_stripped_symbols (System.Native PROGRAMS .)
 endif ()
 
+if (NOT CLR_CMAKE_TARGET_IOS AND NOT CLR_CMAKE_TARGET_TVOS AND NOT CLR_CMAKE_TARGET_ANDROID)
+    set(NATIVE_SOURCES ${NATIVE_SOURCES} entrypoints.c)
```

Comment:
this should probably include CLR_CMAKE_TARGET_BROWSER as well, and we should do the same change on the other System.Native.* libs too

Reply:
Done